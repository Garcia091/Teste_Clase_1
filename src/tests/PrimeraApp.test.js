import React from 'react'
import {shallow} from 'enzyme'
import PrimeraApp from '../PrimeraApp'
import '@testing-library/jest-dom'

describe('pruebas <PrimeraApp/>', () => {
    test('Validar render de <PrimeraApp/>', () => {

        const saludoC = 'Hola clase'
        const subTitulo ='Soy un subtitulo'

        const compo = shallow(
            <PrimeraApp
             saludo={saludoC}
             subtitulo={subTitulo}
              />)
    
        const textParrafo = compo.find('p').text().trim();

        expect(textParrafo).toBe(subTitulo)
        expect(compo).toMatchSnapshot(); 
    })   
})
