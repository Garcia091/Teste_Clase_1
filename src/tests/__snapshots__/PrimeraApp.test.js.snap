// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas <PrimeraApp/> Validar render de <PrimeraApp/> 1`] = `
<Fragment>
  <h1>
     
    Hola clase
     
  </h1>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
